src/operations/vehicle_bringup/vehicle_bringup/node_vehicle_supervisor.py

```diff
@@ -91,8 +91,8 @@ def timer_callback(self):
         self.ros_state.header.stamp = self.get_clock().now().to_msg()
         if (
             self.ros_state.sbg_operational
-            and self.ros_state.lidar_operational
-            and self.ros_state.planning
+            # and self.ros_state.lidar_operational
+            # and self.ros_state.planning
```

Comment:
Just making sure, are we not checking for lidar and planning to confirm g2g?

Reply:
Currently yeah, have some outstanding issues to resolve with diagnostic msgs in general so that'll be a bug fix in the next branch to test on car / at track